mlflow/utils/credentials.py

```diff
@@ -41,3 +41,57 @@ def read_mlflow_creds() -> MlflowCreds:
         username=username_env or username_file,
         password=password_env or password_file,
     )
+
+
+def databricks_login(host, token=None, username=None, password=None):
+    """Log in Databricks and connect MLflow to Databricks workspace.
+
+    This method provides a simple way to connect MLflow to Databricks workspace. Users must specify
+    the host to databricks workspace, and set either `token` or `username`/`password` pair. This
+    method is useful if you want to use Databricks as the tracking server.
+
+    Args:
+        host: string, the URL of the Databricks workspace, must start with "https://", e.g.,
+            "https://community.cloud.databricks.com/".
+        token: string, the Databricks personal access token.
+        username: string, the username of the Databricks workspace, must be used with `password`.
+        password: string, the password of the Databricks workspace, must be used with `username`.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        host = "https://community.cloud.databricks.com/"
+        token = "your-access-token"
+
+        mlflow.login(host, token)
+        with mlflow.start_run() as run:
+            mlflow.log_param("p", 0)
+    """
+    home = os.path.expanduser("~")
+
+    if token is None and (username is None or password is None):
+        raise ValueError(
+            "`username` and `password` must both be set when `token == None`, received "
+            f"`token`={token}, `username`={username}, `password`={password}."
+        )
+    if token is not None and username is not None:
+        raise ValueError(
+            "`username` and `password` must be None when `token != None`, received "
+            f"`token`={token}, `username`={username}, `password`={password}."
+        )
+
+    with open(f"{home}/.databrickscfg", "w+") as f:
```

Comment:
Write mode w+ truncates the file on opening, right? The databrickscfg file is capable of storing many (MANY) profiles in it. I don't think we want to delete it. We want to append an entry with a profile name to it, right?

See: https://docs.databricks.com/en/dev-tools/cli/authentication.html#basic-authentication